test/dotnet/ExpectedOutput/ddl_users.out

```diff
@@ -1,39 +1,54 @@
-#Q#DROP USER IF EXISTS test_usr1
+#Q#DROP USER IF EXISTS test_usr1
 
-#Q#DROP USER IF EXISTS test_usr2
+#Q#DROP USER IF EXISTS test_usr2
 
-#Q#DROP USER IF EXISTS test_usr3
+#Q#DROP USER IF EXISTS test_usr3
 
-#Q#CREATE LOGIN test_pwd1   
-    WITH PASSWORD = '340$Uuxwp7Mcxo7Khy'; 
+#Q#CREATE LOGIN test_pwd1   
+    WITH PASSWORD = '340$Uuxwp7Mcxo7Khy'; 
 
-#Q#CREATE LOGIN test_pwd2   
-    WITH PASSWORD = '340$UuxwpMcxo7Khy';  
+#Q#CREATE LOGIN test_pwd2   
+    WITH PASSWORD = '340$UuxwpMcxo7Khy';  
 
-#Q#CREATE LOGIN test_pwd3   
-    WITH PASSWORD = '340$Uuxwp7Mco7Khy'; 
+#Q#CREATE LOGIN test_pwd3   
+    WITH PASSWORD = '340$Uuxwp7Mco7Khy'; 
 
-#Q#CREATE USER test_usr1 FOR LOGIN test_pwd1;  
+#Q#CREATE USER test_usr1 FOR LOGIN test_pwd1;  
 
-#Q#CREATE USER test_usr2 FOR LOGIN test_pwd2;  
+#Q#CREATE USER test_usr2 FOR LOGIN test_pwd2;  
```

Comment:
why these changes?

Reply:
+1, as these are not really diff, let's remove these unnecessary changes